app/views/candidate_interface/application_form/review.html.erb

```diff
@@ -0,0 +1,75 @@
+<% content_for :title, page_title(:application_form) %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path) %>
+
+<h1 class="govuk-heading-xl">
+  <%= t('page_titles.review_application') %>
+</h1>
+
+<h2 class="govuk-heading-m govuk-!-font-size-27">Courses</h2>
+
+<%= govuk_button_link_to 'TODO: Add a course', '#', class: 'govuk-button--secondary govuk-!-font-size-16 govuk-!-padding-top-1 govuk-!-padding-bottom-1' %>
+
+<h3 class="govuk-heading-m">Personal details</h3>
+
+<section class="app-summary-card govuk-!-margin-bottom-6">
```

Comment:
Again a thing for later, just making a note but we could move this out into a partial as we also have it in `app/views/candidate_interface/personal_details/show.html.erb` which is the review page for personal details.